src/lib/src/hero/hero.component.ts

```diff
@@ -0,0 +1,46 @@
+import { Component, Input, OnInit, ElementRef, ViewChild } from '@angular/core';
+
+@Component({
+  moduleId: module.id,
+  selector: 'prx-hero',
+  styleUrls: ['hero.component.css'],
+  template: `
+    <div #refHero class="hero banner" [class.orange]="orange" [class.blue]="blue">
+      <section>
+        <ng-content select=".hero-title"></ng-content>
+      </section>
+    </div>
+    <div class="hero toolbar" [class.affix]="affixed" (window:scroll)="onScroll()">
```

Comment:
Wondering if you can fix a pre-existing bug here, while you're at it...

- Start with your browser window narrow (we have to assume here that people aren't resizing their browser a whole bunch ... since `affixedY` is only set when this is initialized).
- Scroll down until toolbar affixes
- Toolbar ends up just below the top of the page ... I think due to the header having gotten a bit smaller, due to a media query?